### DISCUSSION TITLE
BooleanColumn searchable

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#6770] [Help]
I am using a boolean column in a table referring to database field that contains either 1 or 0.

`BooleanColumn::make('sent')
	->label('Sent')
	->searchable(isIndividual: true)
	->sortable()
`

The [searchable](https://filamentphp.com/docs/2.x/tables/columns/getting-started#searching) parameter renders an input field on top of the table which is fine for text fields. Is it possible to render some sort of toggle or switch instead?

**Comment by danharrin** [comment]
We have filters for this instead 👍
